modules/installing-ocp-agent-disk-partition.adoc

```diff
@@ -0,0 +1,103 @@
+
+The use of a separate partition for the `/var` directory or a subdirectory of `/var` also prevents data growth in the partitioned directory from filling up the root file system.
+
+The following procedure sets up a separate `/var` partition by adding a machine config manifest that is wrapped into the Ignition config file for a node type during the preparation phase of an installation.
+
+.Procedure
+
+. On your installation host, change to the directory that contains the {product-title} installation program and generate the Kubernetes manifests for the cluster:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory>
+----
+
+. Create a Butane config that configures the additional partition. For example, name the file `$HOME/clusterconfig/98-var-partition.bu`, change the disk device name to the name of the storage device on the `worker` systems, and set the storage size as appropriate. This example places the `/var` directory on a separate partition:
++
+[source,yaml,subs="attributes+"]
+----
+variant: openshift
+version: {product-version}.0
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 98-var-partition
+storage:
+  disks:
+  - device: /dev/<device_name> <1>
+    partitions:
+    - label: var
+      start_mib: <partition_start_offset> <2>
+      size_mib: <partition_size> <3>
+  filesystems:
+    - device: /dev/disk/by-partlabel/var
+      path: /var
+      format: xfs
+      mount_options: [defaults, prjquota] <4>
+      with_mount_unit: true
+----
++
+<1> The storage device name of the disk that you want to partition.
+<2> When adding a data partition to the boot disk, a minimum offset value of 25000 mebibytes is recommended. The root file system is automatically resized to fill all available space up to the specified offset. If no offset value is specified, or if the specified value is smaller than the recommended minimum, the resulting root file system will be too small, and future reinstalls of {op-system} might overwrite the beginning of the data partition.
+<3> The size of the data partition in mebibytes.
+<4> The `prjquota` mount option must be enabled for filesystems used for container storage.
++
+[NOTE]
+====
+When creating a separate `/var` partition, you cannot use different instance types for compute nodes, if the different instance types do not have the same device name.
+====
+
+. Create a manifest from the Butane config and save it to the `clusterconfig/openshift` directory. For example, run the following command:
++
+[source,terminal]
+----
+$ butane $HOME/clusterconfig/98-var-partition.bu -o $HOME/clusterconfig/openshift/98-var-partition.yaml
+----
+
+. Create the Ignition config files:
```

Comment:
At this point we've reached the UPI-specific portion of the docs. What we really want here is to carry on with the rest of the installation by the usual method:

- UPI: `openshift-install create ignition-configs`
- IPI: `openshift-install create cluster`
- Agent: `openshift-install agent create image`